_includes/js/custom.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Colors taken and slightly modified from the Just the Docs theme to match
+// the rest of the site.
+let SUCCESS_COLOR = '#7253ed66';
+let FAILURE_COLOR = '#f5f6fa';
+
+window.onload = () => {
+  let successes = document.body.getElementsByClassName('success-table-element');
+  Array.prototype.forEach.call(successes, element => {
+    element.parentElement.style.backgroundColor = SUCCESS_COLOR;
```

Comment:
I'd prefer for this to set a CSS class on the parent element instead of directly modifying the element style. That will be more flexible. For example, we can share theme colors between CSS styles during development or support theme switching (either built-in to the site itself or via user styles).

roughly like this:

```js
// custom.js
element.parentElement.classList.add("success-table-cell");
```

```scss
// custom.scss
.success-table-cell {
  backgroundColor: '#7253ed66';  // or $success-background-color
}
```

Reply:
Good idea! This actually lets me use the theme variables directly instead of copying their values :)